synfig-studio/src/gui/resources/ui/vectorizer_settings.glade

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.38.2 -->
+<interface>
+  <requires lib="gtk+" version="3.24"/>
```

Comment:
"sadly" we point to a lower gtk version.
Please change to gtk 3.12 on Glade, see if Glade complains about it.
If not, ok. Otherwise comment here about it.

Don't save/commit the file with this change yet though.